### PR TITLE
add ndk support. Fixes #66.

### LINKED ISSUE #66: [Feature] Add android ndk support.
I may have time this weekend.

### PR DESCRIPTION
**Pull request by gengjiawen**
Tested in docker `reactnativecommunity/react-native-android`.
```
  System:
    OS: Linux 4.4 Debian GNU/Linux 9 (stretch) 9 (stretch)
    CPU: (36) x64 Intel(R) Xeon(R) Platinum 8124M CPU @ 3.00GHz
    Memory: 4.93 GB / 68.70 GB
    Container: Yes
    Shell: 4.4.12 - /bin/bash
  Binaries:
    Node: 10.14.1 - /usr/bin/node
    Yarn: 1.12.3 - /usr/bin/yarn
    npm: 6.4.1 - /usr/bin/npm
  Utilities:
    Git: 2.11.0 - /usr/bin/git
  SDKs:
    Android SDK:
      API Levels: 27
      Build Tools: 27.0.3
      System Images: android-19 | Google APIs ARM EABI v7a
      Android NDK: 17.2.4988734
  Languages:
    Bash: 4.4.12 - /bin/bash
    Java: 1.8.0 - /usr/bin/javac
    Perl: 5.24.1 - /usr/bin/perl
    Python: 2.7.13 - /usr/bin/python
  Databases:
    SQLite: 3.22.0 - /opt/android/platform-tools/sqlite3
```

### CODE REVIEW

File: src/packages/get-ndk.js [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-
 function getNdkVersionFromPath(ndkDir) {
-  const metaPath = path.join(ndkDir, 'source.properties')
+  const metaPath = path.join(ndkDir, 'source.properties');
   if (fs.existsSync(metaPath)) {
     const contents = fs.readFileSync(metaPath).toString();
-    const split = contents.split("\n");
-    for (const i of split) {
-      const splits = i.split('=');
+    const split = contents.split('\n');
```
Comment (tabrindle) [inline src/packages/get-ndk.js]: I think we could probably simplify this quite a bit using a regex with a capture group. 

something like this: `/Pkg.Revision = (\d+\.[\d+|.]+)/g`

Comment (gengjiawen) [reply src/packages/get-ndk.js]: Maybe later, the file is literally two lines.

File: src/helpers.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ const osName = require('os-name');
 const path = require('path');
 const packages = require('./packages');
 const utils = require('./utils');
+const { getNdk } = require('./packages/get-ndk');
```
Comment (tabrindle) [inline src/helpers.js]: I dont think this needs to be in a separate file. Helpers are meant to be exportable as a single function from a single place for now.

**Comment by tabrindle** [comment]
If I get a chance, I will pull this down to try it out. 

Thanks so much for the contribution!

File: src/helpers.js (hunk 2 above)
Comment (gengjiawen) [reply src/helpers.js]: I actually write a test case for this function. I am actually thinking moving out all function in `helper` to make every function more testable and the helper function not that crowded.

Comment (gengjiawen) [reply src/helpers.js]: I think this can be our first try.

**Comment by gengjiawen** [comment]
Thanks for trying. I also test it on my win10. Also thinking add android related file to appveyor.

File: src/helpers.js (hunk 2 above)
Comment (gengjiawen) [reply src/helpers.js]: I will put it back tomorrow.

**Pull request merged**